managed/devops/bin/yb_platform_util_py2.py

```diff
@@ -0,0 +1,498 @@
+
+    :param universe_data: Stored universe data.
+    :param universe_name: Name of the new universe to be created.
+    :return: Configured universe json.
+    """
+    configure_json = {}
+    clusters = universe_data['universeDetails']['clusters']
+    user_az_selected = universe_data['universeDetails']['userAZSelected']
+
+    excluded_keys = [
+        "uuid",
+        "storageType",
+        "awsArnString",
+        "useHostname",
+        "preferredRegion",
+        "regions",
+        "index",
+        "placementInfo"
+    ]
+
+    clusters_list = get_cluster_list(clusters, excluded_keys, universe_name)
+    configure_json["clusters"] = clusters_list
+    configure_json["clusterOperation"] = "CREATE"
+    configure_json["userAZSelected"] = user_az_selected
+    configure_json["currentClusterType"] = "PRIMARY"
+
+    return configure_json
+
+
+def get_cluster_list(clusters, excluded_keys, universe_name):
+    """
+    Helper method for creating universe config which returns modified cluster list.
+
+    :param clusters: List of clusters.
+    :param excluded_keys: Keys to be excluded
+    :param universe_name: Name of the new universe to be created.
+    :return: cluster list.
+    """
+    clusters_list = []
+    for each_cluster in clusters:
+        cluster = {}
+        for key, val in each_cluster.items():
+            if key not in excluded_keys:
+                user_intent = {}
+                if key == "userIntent":
+                    if universe_name:
+                        val["universeName"] = universe_name
+                    for key1, val1 in val.items():
+                        if key1 not in excluded_keys:
+                            if key1 == "deviceInfo":
+                                val1.pop("diskIops")
+                                val1.pop("storageType")
+                            user_intent[key1] = val1
+                    cluster[key] = user_intent
+                else:
+                    cluster[key] = val
+        clusters_list.append(cluster)
+    return clusters_list
+
+
+def modify_universe_config(file_name, universe_name=""):
+    """
+    Modify the universe json with new name.
+
+    :param file_name: Name of the json file.
+    :param universe_name: New universe name.
+    :return: Modified universe config data.
+    """
+    data = {}
+    with open(file_name) as f:
+        data = convert_unicode_json(json.loads(f.read()))
+
+    clusters = data["clusters"]
+    if universe_name:
+        for each_cluster in clusters:
+            each_cluster["userIntent"]["universeName"] = universe_name
+        with open(file_name, 'w') as file_obj:
+            json.dump(data, file_obj)
+    return data
+
+
+@exception_handling
+def post_universe_config(configure_json, base_url, customer_uuid, auth_uuid):
+    """
+    Call the universe config URL with the updated data.
+
+    :param configure_json: Universe config json.
+    :param base_url: base url of the platform back end.
```

Comment:
Please use the same style -
Universe ...
**B**ase ...
UUID ...
**A**uthentication ...

Reply:
Please update this here and in other functions too.